beo.js

```diff
@@ -38,35 +66,49 @@ function build () {
     router.on(url, function handle (params, req, res) {
       const handle = node[req.method]
 
-      if (!handle) {
-        res.statusCode = 404
-        res.end()
+      if (req.method === 'GET') {
+        handleNode(handle, params, req, res, null)
+      } else if (req.method === 'POST') {
+        jsonParser(req, function (err, body) {
+          if (err) throw err
+          handleNode(handle, params, req, res, body)
+        })
```

Comment:
this function definition is hoisted, which means it will be allocated even if we do not use it. Move this body-parsing thing into its own function.